### PR TITLE
Update dev with test changes

### PR DESCRIPTION
**Pull request by darrenhunt2**
This pull request will update dev with recent updates to the calendar repository from PR #8. In short, the newest changes include:

- Automatic deployment with Github Actions. Deployment to the test stage on pushes to the test branch now works.
- The README has been updated to include example screenshots of the calendar and more specific deployment instructions.

After these changes are reviewed and finalized, the remaining tasks for updating this repository include:
- Add an architecture diagram to the README (@timbeccue).
- Add a license to the README/repository.
- Write tests to run before deployment with Github Actions, and update the documentation with test information.

We'll have to test that the automatic deployment on pushes to the dev and main branches works once we merge with those branches.

This PR can also close #1, #3, #4, #5, #6, #7.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,385 +1,447 @@
...
   "name": "calendar-backend",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "calendar-backend",
-      "version": "0.1.0",
+      "version": "0.1.1",
```
Comment (kciurleo) [inline package-lock.json:9]: Do we have any sort of guidelines for when to increase the version number?

**Review by kciurleo — COMMENTED** [review]
Looks great to me - the readme and the comments in handler.py especially are extremely helpful!

File: package-lock.json (hunk 1 above)
Comment (darrenhunt2) [reply package-lock.json:9]: I don't think so, but we should figure that out. I felt there were enough changes to change the version number, but we should standardize that!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+### Deployment
+
+Changes pushed to the test, dev, and main branches are automatically deployed to the corresponding test, dev, and production stages with Github Actions. For manual deployment on your local machine, you'll need to fill out the `public_key` and `secrets.json` with the required information. Then, run:
+
+```
+npm install
+serverless plugin install --name serverless-python-requirements
+serverless deploy --stage {stage}
+```
```
Comment (timbeccue) [inline README.md:47]: I would separate these steps.`npm install` and the plugin installations only need to be run once. After that, the user will only need to worry about `serverless deploy...`.

File: package-lock.json (hunk 1 above)
Comment (timbeccue) [reply package-lock.json:9]: This is something that will be part of the developer guidelines doc! I think the convention we should follow is semantic versioning: https://semver.org/

**Review by timbeccue — COMMENTED** [review]
Very nice work! And impressive at 83 commits 😂

Changes all look good, but we should be careful before merging:
- since the github actions are working, merging will trigger a deployment
- the calendar table name will change, which will temporarily break the calendar because all existing events will be gone.
- we need to back up the old table and merge it into one of the new tables as soon as they are deployed
- the frontend and the site code (I think) both use calendar.photonranch.org/dev, so probably should do the above migration into the new dev environment rather than prod. That way the frontend and site code both continue to work as normal.

File: README.md (hunk 2 above)
Comment (darrenhunt2) [reply README.md]: Updated this in the readme

**Pull request merged**